getusermedia.html

```diff
@@ -337,7 +337,7 @@ <h2>{{MediaStream}}</h2>
           <p>If <var>track</var> is not in <var>stream's</var> [=track set=], then abort these steps.</p>
         </li>
         <li>
-          <p>[=MediaStream/Remove a track|Remove=] <var>track</var> from <var>stream</var>'s [=track set=].</p>
+          <p>Remove <var>track</var> from <var>stream</var>'s [=track set=].</p>
```

Comment:
```suggestion
          <p>[=set/Remove=] <var>track</var> from <var>stream</var>'s [=track set=].</p>
```
(make explicit this is a reference to the abstract operation of removing an item from a set by linking to the infra definition)

Reply:
Thanks,
I must admit I thought about it, but refrained from doing so since all such operations would need to be updated, for instance the "add" at the algo right above, but also in many other places in the doc. I thought it's not needed to be *that* verbose, but if you wish I can commit at least this change.

Reply:
yes, it could be applied in more places; but since there is a specific ambiguity your change fixes around the term "remove", I thought this was a good opportunity to start being more explicit